README.md

```diff
@@ -82,33 +84,30 @@ Contribution can be verified using command:
 
 `grunt eslint`
 
-## Deployment
-
-@todo
+## Usage
 
-## Built With
-
-@todo
+Refer to [samsung.github.io/TAU](https://samsung.github.io/TAU/) for library usage instructions
 
 ## Contributing
 
-@todo
-Please read [CONTRIBUTING.md] for details on our code of conduct, and the process for submitting pull requests to us.
+Please read [CONTRIBUTING.md](https://github.com/Samsung/TAU/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
 
 ## Versioning
 
-@todo
+You can find releases in the releases [tab](https://github.com/Samsung/TAU/releases)
+
+Each release number is composed from two parts: 
+
+**Tizen_version_number.TAU_version_for_given_Tizen_release.**
+
+E.g. in case of TAU for Tizen 5.0: 5.0.0...5.0.10, 5.0.11...
```

Comment:
It makes sense to change this to 4.0 ?

Reply:
this is original README from master. I would like to discuss versioning separately from this PR